app/assets/stylesheets/base/typography.scss

```diff
@@ -2,17 +2,18 @@
 @import '../config/import';
 
 :root {
-  --content-rythm: var(--su-2);
-  --content-font-size: var(--fs-l);
+  --content-rhythm: var(--su-2);
+  --content-font-size: var(--fs-base);
```

Comment:
So this basically reduces size of the post font on mobile breakpoint.. Is that intentional?? I kinda liked it being a bit bigger than `base` since it was easier to read and gave enough "importance" to the article text itself..

Reply:
When you pick this up and work on this, can you take a screenshot of what it is using `--fs-base` vs. `--fs-l` and paste the 2 images here? We can open each image on our mobile device and "grock" to see what's feeling okay.